src/Types/index.tsx

```diff
@@ -252,6 +252,9 @@ export enum IconFont {
   Bell = 'bell',
   BellRinging = 'bell-ringing',
   BellSolid = 'bell-solid',
+  Book = 'nav-books-blank',
```

Comment:
Should probably be consistent with whether these are singular "Book" or plural "Books" for the sake of predictability.